### PR TITLE
Add missing information about layouts and activation

### PR DESCRIPTION
**Pull request by Vic-Cooper**
**Select the type of change:**

- [ ] Small Changes - Typos, formatting, slight revisions
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site and Docusaurus

**Purpose of the Pull Request:**
Add missing information about layouts, activating and deactivating a virtual player. Made version compatibility, and entities integration more visible.

**Issue Number:**
[DOCF-5428](https://jira.unity3d.com/browse/DOCF-5428)

### CODE REVIEW

File: mppm/about.md
Comment (kpurcellunity) [inline mppm/about.md:1]: Should we add a section about how Multiplayer Play Mode 1.1.0 is only compatible with Unity Editor versions 6000.0.3f1 or higher. As the minimum version has increased

File: mppm/virtual-players/virtual-players-layout.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+You can display and use the following parts of the [Unity Interface](https://docs.unity3d.com/Manual/UsingTheEditor.html) in a Virtual Player:
+- [Console](https://docs.unity3d.com/Manual/Console.html)
+- [Game](https://docs.unity3d.com/Manual/GameView.html)
+- [Hierarchy](https://docs.unity3d.com/Manual/Hierarchy.html)
+- [Inspector](https://docs.unity3d.com/Manual/UsingTheInspector.html)
+- [Scene](https://docs.unity3d.com/Manual/UsingTheSceneView.html)
+
+note:::
+When you exit Play Mode in the Editor, Unity disables the Hierarchy, Inspector, and Scene views in the virtual player window.
```
Comment (kpurcellunity) [inline mppm/virtual-players/virtual-players-layout.md:15]: Since people typically launch the virtual players starting in edit mode and usually enter playmode , maybe we should reword this to say that Hierarcy , Inspector and Scene View are only available when you enter playmode or are in playmode.

File: mppm/troubleshoot/debug-mppm.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ description: Learn how to test a player running in Multiplayer Play Mode
 
 Use the Test Runner included in the [Unity Test Framework (UTF)](https://docs.unity3d.com/Packages/com.unity.test-framework@2.0/manual/index.html) to test players in a multiplayer project.
 
+Note: Multiplayer Play Mode is compatibly with Unity Editor versions 2023.1 or later.
+
```
Comment (kpurcellunity) [inline mppm/troubleshoot/debug-mppm.md:10]: Again maybe we should note mppm 1.1.0 (latest) minimum editor version is now 6000.0.3f1

**Review by kpurcellunity — CHANGES_REQUESTED** [review]

File: mppm/virtual-players/virtual-players-layout.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+title: Change the layout of a Virtual Player
+description: Learn how to change the layout of a Vrtual Player window.
+---
+
+You can display and use the following parts of the [Unity Interface](https://docs.unity3d.com/Manual/UsingTheEditor.html) in a Virtual Player:
+- [Console](https://docs.unity3d.com/Manual/Console.html)
+- [Game](https://docs.unity3d.com/Manual/GameView.html)
+- [Hierarchy](https://docs.unity3d.com/Manual/Hierarchy.html)
+- [Inspector](https://docs.unity3d.com/Manual/UsingTheInspector.html)
+- [Scene](https://docs.unity3d.com/Manual/UsingTheSceneView.html)
```
Comment (kpurcellunity) [inline mppm/virtual-players/virtual-players-layout.md:12]: Can we also add here that the PlayMode Tools window is available also both in and out of playmode if they have the netcode for entities package installed and then we can link to the documentation we put about that window

**Review by kpurcellunity — CHANGES_REQUESTED** [review]

File: mppm/about.md
Comment (Vic-Cooper) [reply mppm/about.md:1]: This PR is for the 1.0.0 version, not 1.1.0. We'll add that compatability info once that version of the docs is released, I'm just waiting on release notes.

File: mppm/troubleshoot/debug-mppm.md (hunk 2 above)
Comment (Vic-Cooper) [reply mppm/troubleshoot/debug-mppm.md:10]: See other reply.

Comment (Vic-Cooper) [reply mppm/troubleshoot/debug-mppm.md:10]: Will add this update to 1.1.0 when released :)

File: mppm/about.md
Comment (Vic-Cooper) [reply mppm/about.md:1]: Will add this update to 1.1.0 when released :)

File: mppm/about.md [hunk 4]
```diff
@@ -6,6 +6,9 @@ description: Overview of Multiplayer Play Mode
 
 Use Multiplayer Play Mode to test multiplayer functionality within the Unity Editor. You can simulate up to four players (the Main Editor and three Virtual Players) simultaneously on the same development device while using the same source assets on disk. Multiplayer Play Mode can help you create multiplayer development workflows that reduce project build times, run it locally, and test the server-client relationship.
 
+## Compatibility
```
Comment (suearkinunity) [inline mppm/about.md:9]: General question: does it render correctly when you don't have a blank line after a header/before a list/between paragraphs?

File: mppm/about.md [hunk 5]
```diff
@@ -6,6 +6,9 @@ description: Overview of Multiplayer Play Mode
...
+Multiplayer Play Mode version 1.1.0 is compatible with Unity Editor versions 6000.0.3f1 or later.
```
Comment (suearkinunity) [inline mppm/about.md:10]: 1.1.0 is very limited; can we say "1.1 and later"?

File: mppm/install.md [hunk 6]
```diff
@@ -4,10 +4,11 @@ title: Install Multiplayer Play Mode
 description: How to install Multiplayer Play Mode
 ---
 
-To begin testing your multiplayer functionality, you must first install the Multiplayer Play Mode package.
+You can use Multiplayer Play Mode in Unity Editor versions 2023.1 or later.
```
Comment (suearkinunity) [inline mppm/install.md:7]: You specified 1.1.0 earlier; I like this more generic version better, but should they be consistent?

File: mppm/install.md [hunk 7]
```diff
@@ -4,10 +4,11 @@ title: Install Multiplayer Play Mode
...
+To install the Multiplayer Play Mode package, do the following.
```
Comment (suearkinunity) [inline mppm/install.md:8]: Not a fan of "do the following", and there's no colon (should there be?). But that's more of a personal thing.

File: mppm/troubleshoot/debug-mppm.md [hunk 8]
```diff
@@ -6,6 +6,10 @@ description: Learn how to test a player running in Multiplayer Play Mode
 
 Use the Test Runner included in the [Unity Test Framework (UTF)](https://docs.unity3d.com/Packages/com.unity.test-framework@2.0/manual/index.html) to test players in a multiplayer project.
 
+:::note: 
+Multiplayer Play Mode is compatible with Unity Editor versions 2023.1 or later.
```
Comment (suearkinunity) [inline mppm/troubleshoot/debug-mppm.md:10]: Same question about 1.1.0/more generic as above.

File: mppm/virtual-players/virtual-players-layout.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
+---
+id: virtual-players-layout
+title: Change the layout of a Virtual Player
+description: Learn how to change the layout of a Vrtual Player window.
```
Comment (suearkinunity) [inline mppm/virtual-players/virtual-players-layout.md:4]:
```suggestion
description: Learn how to change the layout of a Virtual Player window.
```

File: mppm/virtual-players/virtual-players-layout.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+You can display and use the following parts of the [Unity Interface](https://docs.unity3d.com/Manual/UsingTheEditor.html) in a Virtual Player:
```
Comment (suearkinunity) [inline mppm/virtual-players/virtual-players-layout.md:7]: Do you need a blank line between the paragraph and the list for it to render correctly?

Comment (suearkinunity) [reply mppm/virtual-players/virtual-players-layout.md:7]: How about something like "The Virtual Player can display the following parts of the Unity Interface"? That you can use them once they're displayed is, I think, obvious.

File: mppm/virtual-players/virtual-players-layout.md [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+- [Console](https://docs.unity3d.com/Manual/Console.html)
+- [Game](https://docs.unity3d.com/Manual/GameView.html)
+- [Hierarchy](https://docs.unity3d.com/Manual/Hierarchy.html)
+- [Inspector](https://docs.unity3d.com/Manual/UsingTheInspector.html)
+- [Scene](https://docs.unity3d.com/Manual/UsingTheSceneView.html)
+- [PlayMode Tools](https://docs.unity3d.com/Packages/com.unity.netcode@1.2/manual/playmode-tool.html). This Window only appears in projects that use Netcode for Entities.
+
+:::note
+You can only use the Hierarchy, Inspector, and Scene windows when you enter Play Mode.
```
Comment (suearkinunity) [inline mppm/virtual-players/virtual-players-layout.md:16]:
```suggestion
You can only use the Hierarchy, Inspector, and Scene windows in Play Mode.
```

File: mppm/virtual-players/virtual-players-layout.md [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+:::
+
+## Display a window in a virtual player
+
+To display a window in a Virtual Player, perform the following actions: 
+
+1. [Activate](../virtual-players/virtual-players-enable) a Virtual Player.
+2. Select the **Layout** drop down.
```
Comment (suearkinunity) [inline mppm/virtual-players/virtual-players-layout.md:24]:
```suggestion
2. Select the **Layout** dropdown.
```

Comment (suearkinunity) [reply mppm/virtual-players/virtual-players-layout.md:24]: https://docs-style-guide.unity.com/terminology/user-interface/ui-symbols dropdown is a single word

File: mppm/virtual-players/virtual-players-layout.md [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+3. Check the box next to the window(s) you want to appear in this Virtual Player.
```
Comment (suearkinunity) [inline mppm/virtual-players/virtual-players-layout.md:25]:
```suggestion
3. Select the box next to the window(s) you want to appear in this Virtual Player.
```

Comment (suearkinunity) [reply mppm/virtual-players/virtual-players-layout.md:25]: Can it be "you want to add to this Virtual Player"?

File: mppm/virtual-players/virtual-players-layout.md [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+4. Select **Apply**.
+
+## Hide a window in a virtual player
+
+To hide a window in a Virtual Player, perform the following actions: 
+
+1. [Activate](../virtual-players/virtual-players-enable) a Virtual Player.
+2. Select the **Layout** drop down.
```
Comment (suearkinunity) [inline mppm/virtual-players/virtual-players-layout.md:33]:
```suggestion
2. Select the **Layout** dropdown.
```

File: mppm/virtual-players/virtual-players-layout.md [hunk 15]
```diff
@@ -0,0 +1,35 @@
...
+3. Check the box next to the window(s) you want to appear in this Virtual Player.
+4. Select **Apply**.
+
+## Hide a window in a virtual player
+
+To hide a window in a Virtual Player, perform the following actions: 
+
+1. [Activate](../virtual-players/virtual-players-enable) a Virtual Player.
```
Comment (suearkinunity) [inline mppm/virtual-players/virtual-players-layout.md:32]: "a" virtual player, or is it "the" at this point, because it already exists?

File: mppm/virtual-players/virtual-players-layout.md [hunk 16]
```diff
@@ -0,0 +1,35 @@
...
+2. Select the **Layout** drop down.
+3. Select the checked box next to the window(s) you want to hide in this Virtual Player.
```
Comment (suearkinunity) [inline mppm/virtual-players/virtual-players-layout.md:34]:
```suggestion
3. Deselect the checked box next to the window(s) you want to hide in this Virtual Player.
```

Comment (suearkinunity) [reply mppm/virtual-players/virtual-players-layout.md:34]: https://docs-style-guide.unity.com/terminology/terminology-list#d

File: mppm_versioned_docs/version-1.0.0/about.md [hunk 17]
```diff
@@ -6,6 +6,9 @@ description: Overview of Multiplayer Play Mode
 
 Use Multiplayer Play Mode to test multiplayer functionality within the Unity Editor. You can simulate up to four players (the Main Editor and three Virtual Players) simultaneously on the same development device while using the same source assets on disk. Multiplayer Play Mode can help you create multiplayer development workflows that reduce project build times, run it locally, and test the server-client relationship.
 
+## Compatibility
```
Comment (suearkinunity) [inline mppm_versioned_docs/version-1.0.0/about.md:9]: Do you need a blank line?

File: mppm_versioned_docs/version-1.0.0/about.md [hunk 18]
```diff
@@ -6,6 +6,9 @@ description: Overview of Multiplayer Play Mode
...
+Multiplayer Play Mode version 1.0.0 is compatible with Unity Editor versions 2023.1 or later.
```
Comment (suearkinunity) [inline mppm_versioned_docs/version-1.0.0/about.md:10]: Same question as the previous time the line appeared.

**Review by suearkinunity — CHANGES_REQUESTED** [review]
Everything appears twice; I left comments only for the first time things appear, but they apply to the second appearance.

**Review by kpurcellunity — APPROVED** [review]

File: mppm/about.md (hunk 5 above)
Comment (Vic-Cooper) [reply mppm/about.md:11]: Let's check cieht @kpurcellunity

File: mppm/about.md (hunk 4 above)
Comment (Vic-Cooper) [reply mppm/about.md:9]: I think the space is necessary but I was honestly just fitting the convention of the existing docs - I'll test it.

File: mppm/install.md (hunk 6 above)
Comment (Vic-Cooper) [reply mppm/install.md:7]: I see what you mean. I worded it like this because version 1.0.0  is generically compatible, however, version 1.1.0 is _only_ compatible with that specific Unity version. So I actually need to update this line to match the other one.

File: mppm/install.md (hunk 7 above)
Comment (Vic-Cooper) [reply mppm/install.md]: Oh there should definitely be a colon, my mistake. 
As for "do the following" - that's just to make it a clause, but I can replace it with "follow these steps" as in the style guide.

File: mppm/virtual-players/virtual-players-layout.md (hunk 16 above)
Comment (Vic-Cooper) [reply mppm/virtual-players/virtual-players-layout.md:34]: Thank you!

File: mppm/troubleshoot/debug-mppm.md (hunk 8 above)
Comment (Vic-Cooper) [reply mppm/troubleshoot/debug-mppm.md:10]: Just double-check with @kpurcellunity: 

Should we always specify versions 6000.0.3f1 or later in 1.1.0 docs?

File: mppm/virtual-players/virtual-players-layout.md (hunk 15 above)
Comment (Vic-Cooper) [reply mppm/virtual-players/virtual-players-layout.md:32]: Unity creates the virtual player when you activate it - it doesn't exist until you do that so "a" is correct :)

**Comment by Vic-Cooper** [comment]
Resolved feedback from Sue, thank you :) 
Just waiting on @kpurcellunity 's reply regarding version numbers.

File: mppm/troubleshoot/debug-mppm.md (hunk 8 above)
Comment (Vic-Cooper) [reply mppm/troubleshoot/debug-mppm.md:10]: Yes we do need to be specific to the unity 6 version, I'll update this.

File: mppm/virtual-players/virtual-players-enable.md [hunk 19]
```diff
@@ -1,22 +1,36 @@
...
 id: virtual-players-enable
-title: Enable virtual players
+title: Activate a Virtual Player
 description: Enable up to 3 additional Virtual Players to your Main Editor Player in Multiplayer Play Mode.
 ---
-:::important
-You can only attach **Player Tags** to inactive players (including the **Main Editor Player**). If you plan to use **Player Tags**, [create and configure your tags](../player-tags/player-tags-create.md) before you enable virtual players.
+
+To use a Virtual Player to test multiplayer gameplay, activate at least one Virtual Player.
+
```
Comment (kpurcellunity) [inline mppm/virtual-players/virtual-players-enable.md:8]: Can we clarify that this folder is only created the first time you launch a virtual player

Comment (kpurcellunity) [reply mppm/virtual-players/virtual-players-enable.md:8]: If you delete the folder it will also create a new one, or if you update mppm if should clear these folders as well. Otherwise it will use the folder it has cached.

File: mppm/virtual-players/virtual-players-enable.md [hunk 20]
```diff
@@ -1,22 +1,36 @@
...
+4. When active, the Virtual Player(s) appear in Play Mode in the Main Editor window. A separate window opens for each enabled player with the player name as the window title.
 
 To understand how **Multiplayer Play Mode** works with **Play mode**, refer to [**Synchronize with Play mode**](../sync-play-mode.md).
+
+## Deactivate a Virtual Player
+
+Virtual players do not automatically deactivate when you close the Virtual Player window. 
+To deactivate a **Virtual Player**.
+
+1. In the Unity Editor, open the Multiplayer Play Mode window by navigating to **Window** > **Multiplayer Play Mode**.
```
Comment (kpurcellunity) [inline mppm/virtual-players/virtual-players-enable.md:35]: Clarify deactivating the VP does not delete the Virtual Player's folder in the VP. Deactivating only stops the virtual players process

**Review by kpurcellunity — CHANGES_REQUESTED** [review]

**Review by kpurcellunity — APPROVED** [review]
looks great :)

**Review by suearkinunity — APPROVED** [review]
